dev/storage_base.h

```diff
@@ -728,6 +741,17 @@ namespace sqlite_orm {
                 }
             }
 
+            // migration internal API
+            void start_migration() {
+                this->begin_exclusive_transaction();
+            }
+            void commit_migration() {
+                this->commit();
+            }
+            void abort_migration() {
+                this->rollback();
```

Comment:
just a question: why don't you call `rollback` instead of `abort_migration`? These functions are not migrations, these functions are aliases for transactions actually

Reply:
well now they have a little more substance so I think we should keep them.. please check them out and look at backup_table()'s implementation...